Run error on Windows 10, branch master.

Everything seemed to be ok when I was still in branch 'Add windows support', then I saw the branch had been merged to master, I cloned the app again and built it without any errors, but when I ran the app after `npm run pack`, I got this error:

![Error](http://i.imgur.com/4F0T5ZQ.png "Error on windows 10")

Windows 10, branch master, latest code.

if this helps, i got this error message upon trying to start `./hyper` in `linux-unpacked`:

```
Module version mismatch. Expected 50, got 51.
A JavaScript error occurred in the main process
Uncaught Exception:
Error: `pty.js` failed to load. Typically this means that it was built incorrectly. Please check the `README.me` to more info.
    at createPtyJsError (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/app/session.js:11:32)
    at Object.<anonymous> (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/app/session.js:17:9)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/app/index.js:30:17)
```

this line:
```
Module version mismatch. Expected 50, got 51.
```

looks really suspicious.
node v7.1.0 and npm 3.10.9 btw.

looking in `session.js`, you can find this code:
```js
...
  spawn = require('pty.js').spawn;
} catch (err) {
  throw createPtyJsError();
}
...
```

Adding `console.log(err);` above `createPtyJsError` gives me:

```
Module version mismatch. Expected 50, got 51.
{ Error: Cannot find module '../bin/linux/x64.m50.node'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/app/node_modules/pty.js/lib/pty.js:18:9)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12) code: 'MODULE_NOT_FOUND' }
A JavaScript error occurred in the main process
Uncaught Exception:
Error: `pty.js` failed to load. Typically this means that it was built incorrectly. Please check the `README.me` to more info.
    at createPtyJsError (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/app/session.js:11:32)
    at Object.<anonymous> (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/app/session.js:18:9)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/cheesy/dev/prg/hyper/dist/linux-unpacked/resources/app/index.js:30:17)
```
i have no idea what `x64.m50.node` is. google returns nothing helpful. apparently though, my system can't find it, which i guess is causing the error.

hope this helps.

here it is for windows:

```
{ Error: Cannot find module '..\bin\win32\x64.m49.node'
    at Module._resolveFilename (module.js:440:15)
    at Function.Module._resolveFilename (C:\Users\aluca\git\hyper\node_modules\electron\dist\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\aluca\git\hyper\app\node_modules\pty.js\lib\pty.js:18:9)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12) code: 'MODULE_NOT_FOUND' }
App threw an error during load
Error: `pty.js` failed to load. Typically this means that it was built incorrectly. Please check the `README.me` to more info.
    at createPtyJsError (C:\Users\aluca\git\hyper\app\session.js:11:32)
    at Object.<anonymous> (C:\Users\aluca\git\hyper\app\session.js:18:9)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\aluca\git\hyper\app\index.js:30:17)
```

--- a/package.json
+++ b/package.json
@@ -94,7 +94,7 @@
     "copy-webpack-plugin": "^4.0.0",
     "cross-env": "3.1.3",
     "electron": "1.4.5",
-    "electron-builder": "^7.11.4",
+    "electron-builder": "8.6.0",
     "electron-devtools-installer": "^2.0.0",
     "eslint-config-xo-react": "^0.10.0",
     "eslint-plugin-react": "^6.3.0",

--- a/package.json
+++ b/package.json
@@ -51,6 +51,11 @@
         "AppImage"
       ]
     },
+    "win": {
+      "target": [
+        "squirrel"
+      ]
+    },
     "mac": {
       "category": "public.app-category.developer-tools"
     }

--- a/test/index.js
+++ b/test/index.js
@@ -12,7 +12,7 @@ test.before(async () => {
 
   switch (process.platform) {
     case 'linux':
-      pathToBinary = path.join(__dirname, '../dist/linux-unpacked/Hyper');
+      pathToBinary = path.join(__dirname, '../dist/linux-unpacked/hyper');
       break;
 
     case 'darwin':

